examples/if_ls_glob.bats

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bats
+
+TEST_DIR="test_dir"
+
+setup() {
+  mkdir "${TEST_DIR}"
+  cd "${TEST_DIR}"
+
+  bats() {
+    true
+  }
+}
+
+teardown() {
+  cd ..
+  rm -rf "${TEST_DIR}"
+}
+
+main() {
+  if ls *.bats >/dev/null 2>&1; then
+    bats -t *.bats
```

Comment:
could you please explain why does the test function need to run tests. this is inception. very confusing.

can this simply create a set of fixture files?

or maybe I'm missing the point about this test example (hence the file comment would help to set the context)

Reply:
have removed the `ls` script for now because it is confusing!